src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -628,14 +666,14 @@ private void CheckStake(Block block, ChainedBlock chainTip)
                 {
                     Index = workerIndex,
                     Logger = this.loggerFactory.CreateLogger(this.GetType().FullName, $"[Worker #{workerIndex}] "),
-                    Coins = new List<StakeTx>(),
+                    utxoStakeDescriptions = new List<UtxoStakeDescription>(),
                     CoinstakeContext = coinstakeContext,
                     Result = workersResult
                 };
 
-                int coinCount = Math.Min(setCoins.Count - coinIndex, CoinsPerCoinstakeWorker);
-                cwc.Coins.AddRange(setCoins.GetRange(coinIndex, coinCount));
-                coinIndex += coinCount;
+                int txesCount = Math.Min(stackingUtxoDescriptions.Count - coinIndex, utxoStakeDescriptionsPerCoinstakeWorker);
```

Comment:
coinCount was better, but feel free to rename to stakingUtxoCount or whatever, just not transactions